cds/modules/previewer/templates/cds_previewer/video/video.html

```diff
@@ -29,21 +29,66 @@
 
 {%- block javascript %}
   {{ super() }}
+  {% if not config.THEOPLAYER_LIBRARY_LOCATION %}
+    {%- set log_media_views_url = config.LOG_USER_ACTIONS_ENDPOINTS.get('media_view', '').format(base_url=config.LOG_USER_ACTIONS_ENDPOINTS.get('base_url', '') , recid=recid) -%}
+    {%- set file_key = file.key -%}
+
+    <script>
+       // function from https://www.w3schools.com/js/js_cookies.asp
+       function getCookie(cname) {
```

Comment:
not sure I understand why we need it

Reply:
it is the same as in RDM for passing the CSRF token on js calls